### ISSUE TITLE
Neo4j connectivity needs some improvement

### ISSUE THREAD

**Issue by hvandenb** [adrianco/spigo#68]
Need the following:
* Improve the NEO4JURL piece to have actual url format
* Set password to the detail neo4j password when not provided
* Write up the document to help with setup

I'll put something together.

**Comment by adrianco** [comment]
I think spigo should start by sending sending a DELETE for just the nodes and edges for the architecture that it's about to write.

There are some interesting queries I need to document as well.

Thanks for starting this issue.

**Comment by hvandenb** [comment]
Agreed we either have to delete the graph if the infrastructure is a whole consistent replacement. Trying to do deltas might more of a Challange. I have the code change in my fork for improving the url etc. including some improved docs.
